scripts/remove/audiobookshelf.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Audiobookshelf
+
+__remove_audiobookshelf() {
+    echo_progress_start "Removing audiobookshelf apt sources"
+    systemctl stop audiobookshelf.service
+    sysctl disable audiobookshelf.service
```

Comment:
`disable` also takes `--now` parameter so you can get these done in one line. Would also be nice to pass the `-q` flag so it's quiet